sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -78,44 +106,44 @@
   /**
    * Retrieve a State based on their key.
    *
-   * @param state        The key of the State to be retrieved
+   * @param state        The key of the State to be retrieved.
    * @param stateOptions
    * @param clazz        the Type of State needed as return.
-   * @param <T>          the Type of the return
-   * @param <K>          The Type of the key of the State
-   * @return A Mono Plan for the requested State
+   * @param <T>          the Type of the return.
+   * @param <K>          The Type of the key of the State.
```

Comment:
K is not needed.

Reply:
stateOptions can be an optional parameter.

Reply:
By optional, you mean adding it to the comment, right? Java does not have a way to specify a default value for a parameter to make it optional.